src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -255,6 +294,9 @@ export default class StyleConfigurator {
       if (bpmnCell.bpmnElement instanceof SequenceFlow) {
         styles.push(bpmnCell.bpmnElement.sequenceFlowKind);
       }
+      if (bpmnCell.bpmnElement instanceof AssociationFlow) {
```

Comment:
Missing unit test for this 😉

Reply:
yes you are right:)
added them in last commit